types/Rule.ts

```diff
@@ -91,3 +94,33 @@ export interface DeleteRuleParams {
 }
 
 export interface DeleteRuleResponse {}
+
+export interface IRulesCache {
+  [key: string]: RuleItemInfoBase[] | [];
+}
+
+export interface IRulesStore {
+  cache: IRulesCache;
```

Comment:
This doesn't need to be public so we can leave it out, no? Similarly for other properties which shouldn't be exposed like the invalidateCache method?

Reply:
public/private done